packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -451,36 +425,118 @@ export const Tabs = (props: TabsProps) => {
     ownerDoc.addEventListener('scroll', determineScrollingPlacement);
   }, [overflowOptions?.useOverflow, determineScrollingPlacement]);
 
-  const onTabClick = React.useCallback(
-    (index: number) => {
-      if (onTabSelected) {
-        onTabSelected(index);
-      }
-      setCurrentActiveIndex(index);
-    },
-    [onTabSelected],
+  return (
+    <Box
+      className={cx(
+        'iui-tabs',
+        `iui-${type}`,
+        {
+          'iui-green': color === 'green',
+          'iui-animated': type !== 'default' && isClient,
+          'iui-not-animated': type !== 'default' && !isClient,
+          'iui-large': hasSublabel,
+        },
+        className,
+      )}
+      data-iui-overflow={overflowOptions?.useOverflow}
+      data-iui-scroll-placement={scrollingPlacement}
+      role='tablist'
+      ref={refs}
+      {...rest}
+    >
+      {items.map((item, index) => {
+        return (
+          <TabsTabListContext.Provider
+            value={{
+              index,
+              focusedIndex,
+              setFocusedIndex,
+              focusActivationMode,
+              hasSublabel,
+              setHasSublabel,
+            }}
+            key={index}
+          >
+            {item}
+          </TabsTabListContext.Provider>
+        );
+      })}
+    </Box>
   );
+}) as PolymorphicForwardRefComponent<'div', TabsTabListOwnProps>;
+TabsTabList.displayName = 'Tabs.TabList';
+
+// ----------------------------------------------------------------------------
+// Tabs.Tab component
+
+type TabsTabOwnProps = {
+  /**
+   * Tab label used for simple Tab construction.
+   * Cannot be used with tabs that have icons or descriptions.
+   */
+  label?: string | React.ReactNode;
+  /**
+   * Flag whether the tab is disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Flag whether the tab is active.
+   * @default 'false'
+   */
+  isActive?: boolean;
+  /**
+   * Callback fired when the isActive prop changes.
+   */
+  onActiveChange?: () => void;
```

Comment:
`onActiveChange` makes it sound like it would also be called when a tab 

what do you think of changing it to `onActivated`/`onActive`?

alternatively, both `isActive`+`onActiveChange` -> `isSelected`+`onSelected`

Reply:
I've changed this prop to `onActivated`